### ISSUE TITLE
1.19.2

### ISSUE THREAD

**Issue by Latenciiii** [kangarko/Boss#1119]
Hello,

Is there a limit on amount of regions you can have within boss? ive been using the boss regions from the start for bosses spawning, however im wondering now, since the update has something changed? whenever i try to add a percentage to a boss spawning in the spawnrule region tab, it says 'click regions icon first' ive never seen that message before, i have even tried to amend it in the config to no avail as it just turns it off again.

Pictures attached to help.

Thank you
![PXL_20221123_143159768](https://user-images.githubusercontent.com/42725852/203573145-e69bd832-8c00-4a39-b390-ad89fdeac017.jpg)
<img width="320" alt="boss1" src="https://user-images.githubusercontent.com/42725852/203573183-a4ab2d9a-d15a-4a5e-a5d7-34c657a1ff46.png">

**Comment by Latenciiii** [comment]
My regions seem enabled as other bosses are spawning when entering regions? 

I dont see the glass icon and when i hover over the beacon it says enabled? 

Sorry to be a pain, this one seems weird though! Not had this before!


<img width="290" alt="boss1" src="https://user-images.githubusercontent.com/42725852/203863462-b01364a1-e9f5-4a3d-857c-5716a443e3a0.png">

**Comment by kangarko** [comment]
This menu I am talking about in Spawn Rule > Regions, have you enabled it here?

<img width="646" alt="Screenshot 2022-11-26 at 09 25 20" src="https://user-images.githubusercontent.com/1918426/204081781-eaf4ea15-2e14-4b56-a7f8-a3d773c12eb4.png">

**Comment by Latenciiii** [comment]
So i believe this is a bug? Ive made 20+ bosses and i choose the spawning feature 'Spawn when a player enters a region' this used to work and i would just increase the amount on the lapis screen. I can now only open the 'Select region type' gui when i choose the 'Randomly around players' spawn rule? 

When i choose the 'player enters region' this doesnt seem to work anymore, i even tried enabling the region on this screen and then coming out of it and try again but it didnt work,

TLDR- Regions only work on (when setting up a new boss) 'randomly spawn around players' - Bosses prior to this still work and spawn how i want them to.

Thanks

**Comment by kangarko** [comment]
Sorry for the massive delay, I have fixed that bug and you can now select region chances for "on enter region" spawn rule!
